linux/debian/postinst

```diff
@@ -1,6 +1,17 @@
 #!/usr/bin/env bash
 echo "restarting ckb-next-daemon"
-sudo systemctl daemon-reload
-sudo systemctl enable ckb-next-daemon.service
-sudo systemctl start ckb-next-daemon.service
-sudo service ckb-next-daemon restart
+if [ -f /usr/bin/systemctl ]; then
+    systemctl daemon-reload
+    systemctl enable ckb-next-daemon.service
+    if [ $? -ne 0 ]; then
+        echo "ckb-next-daemon unit file not found"
+        echo "restart cannot proceed, please check your install is complete."
+        exit 0;
+    fi
+    systemctl start ckb-next-daemon.service
+elif [ -f /usr/sbin/service ]; then
+    service ckb-next-daemon restart
+else
+    echo "no systemctl binary, no service binary, CKB-Next does not support SysV init"
```

Comment:
[That's news to me.](https://github.com/ckb-next/ckb-next/blob/master/linux/sysvinit/ckb-next-daemon.in)